test-cases/gutenberg/writing-flow/splitting-merging.md

```diff
@@ -75,20 +75,3 @@ Start from an empty post.
 - Press enter two times to create an empty block in between 
 - Press back two times to merge everything again.
 - Check that the blocks were merged.
-
-
-##### TC006
-
-**Splitting/merge list block**
-
-- Write some items on a list block
-- Put the caret by the half of them (at the end of the item) and press Enter to create a new item.
-- Press Enter again to split the block into two, with an empty paragraph block in the middle. The paragraph block at the middle should be focused.
```

Comment:
I checked the automated tests in https://github.com/WordPress/gutenberg/pull/45048, and although we cover merging list blocks, I'm not sure we fully cover this test case, especially the splitting part. WDYT?